catalog/spatial/csw/spatial-csw-action/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/actions/DeleteAction.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.spatial.ogc.csw.catalog.actions;
+
+import net.opengis.cat.csw.v_2_0_2.QueryConstraintType;
+
+public interface DeleteAction {
```

Comment:
Could you add some javadoc to this? And should it be marked as experimental?

Reply:
Looking at this further; it might be best to mark them as experimental so we aren't boxed into a corner if we need to make changes

Reply:
I marked all the interfaces and `CswActionTransformerProvider` as experimental